solidity/contracts/lib/interfaces/izeto.sol

```diff
@@ -53,6 +53,7 @@ interface IZeto {
         address indexed submitter,
         bytes data
     );
+    event IdentityRegistered(uint256[2] publicKey, bytes data);
```

Comment:
This won't be used by every Zeto variant, right? Maybe should consider a separate interface.

Also, I've noticed we only define events in the interfaces, and not method calls. It's useful to standardize the method calls as well, especially since there are so many variants. But that doesn't have to be part of this PR.